### ISSUE TITLE
LIBMESH_HAVE_BOOST doesn't use bool

### ISSUE THREAD

**Issue by aeslaughter** [libMesh/libmesh#1365]
I have configure libMesh with an external Boost library, however my libmesh_config.h looks like:

```
/* define if the Boost library is available */
#ifndef LIBMESH_HAVE_BOOST
#define LIBMESH_HAVE_BOOST /**/
#endif
```

For the MOOSE test harness, we need this to be:

```
#define LIBMESH_HAVE_BOOST 1
```

**Comment by jwpeterson** [comment]
On Fri, Jun 16, 2017 at 11:55 AM, Andrew E Slaughter <
notifications@github.com> wrote:

> I have configure libMesh with an external Boost library, however my
> libmesh_config.h looks like:
>
> /* define if the Boost library is available */
> #ifndef LIBMESH_HAVE_BOOST
> #define LIBMESH_HAVE_BOOST /**/
> #endif
>
>
This is really weird, but I think it is caused by this line in
ax_boost_base.m4

        AC_DEFINE(HAVE_BOOST,,[define if the Boost library is available])

The second argument to AC_DEFINE should be 1...


-- 
John

**Comment by jwpeterson** [comment]
This was fixed by #1366.
